src/live-common-set-supported-currencies.js

```diff
@@ -36,4 +36,5 @@ setSupportedCurrencies([
   "bitcoin_testnet",
   "ethereum_ropsten",
   "cosmos_testnet",
+  "elrond",
```

Comment:
also, we shouldn't enable yet before go "ful prod", it would be good to merge the rest of the work to have it land in nightly, without it enabled